packages/core-plugin-api/src/analytics/AnalyticsContext.tsx

```diff
@@ -85,19 +85,19 @@ export const AnalyticsContext = (options: {
 
 /**
  * Returns an HOC wrapping the provided component in an Analytics context with
- * the given values.
+ * the given attributes.
  *
  * @param Component - Component to be wrapped with analytics context attributes
- * @param values - Analytics context key/value pairs.
- * @internal
+ * @param attributes - Analytics context key/value pairs.
+ * @public
  */
-export function withAnalyticsContext<TProps extends {}>(
+export function withAnalyticsContext<TProps extends {} = any>(
   Component: React.ComponentType<TProps>,
-  values: AnalyticsContextValue,
+  attributes: Partial<AnalyticsContextValue>,
 ) {
```

Comment:
Could be nice to add an explicit return type here to clean things up, although we don't really have a precedent for that